src/portfft/descriptor.hpp

```diff
@@ -428,21 +435,27 @@ class committed_descriptor {
 
   /**
    * Sets the implementation dependant specialization constant value
+   * @tparam T scalar type of the committed descriptor
    * @param top_level implementation to dispatch to
    * @param in_bundle input kernel bundle to set spec constants for
    * @param length length of the fft
    * @param factors factors of the corresponsing length
-   * @param multiply_on_load Whether the input data is multiplied with some data array before fft computation.
-   * @param multiply_on_store Whether the input data is multiplied with some data array after fft computation.
+   * @param multiply_on_load Whether the input data is multiplied with some data array before fft computation
+   * @param multiply_on_store Whether the input data is multiplied with some data array after fft computation
    * @param scale_factor_applied whether or not to multiply scale factor
-   * @param level sub implementation to run which will be set as a spec constant.
+   * @param level sub implementation to run which will be set as a spec constant
+   * @param take_conjugate_on_load whether or not to take conjugate of the input
+   * @param take_conjugate_on_store whether or not to take conjugate of the output
+   * @param scale_factor Scale to be applied to the result
    * @param factor_num factor number which is set as a spec constant
-   * @param num_factors total number of factors of the committed size, set as a spec constant.
+   * @param num_factors total number of factors of the committed size, set as a spec constant
    */
+  template <typename T>
```

Comment:
Why not use `Scalar` from the parent class?

Reply:
Genuinely forgot we had access to that,
Fixed